wp-content/themes/core/pcss/content/panel/_wysiwyg.pcss

```diff
@@ -0,0 +1,39 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Panels: WYSIWYG
+ *
+ * ----------------------------------------------------------------------------- */
+
+.site-panel--wysiwyg {
+
+	.s-title {
+		margin-bottom: 0;
+	}
+
+	.s-desc {
+
+		.s-title ~ & {
+			margin-top: var(--spacer-0);
+		}
+	}
+
+	.g-col:not(:last-child) {
+		margin-bottom: var(--spacer-4);
+	}
+}
+
+.s-header {
+	@media (--viewport-medium) {
```

Comment:
Remember that this is a globally available section utility class, so we should stay aware of whether the change we are making should be applicable only to this particular panel (sandbox it in your panel class) or is a global change (port to the sections header partial) and handle accordingly ;)

Fwiw, these do look like global changes.

Reply:
My mistake. I'm not sure the width should be global for this, so scoping to the panel until we see that pattern really emerge.